### PR TITLE
Created  a Contributor's Guideline

### PR DESCRIPTION
**Pull request by Rishikeshrajrxl**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(I have Created a Contributor's Guideline as Per assigned work a few days ago.)
Here I have used images and screenshots for better understanding. This is going to help new geeks who are new to Open Source. 

I have also used Grammarly.
Add Screenshots wherever possible.
Add the basic information about creating Issues, Pull Requests, and Branching.
Mention tutorials related to Git and Github to help them better understand.

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(#542 )

### CODE REVIEW

**Comment by Rishikeshrajrxl** [comment]
@siddharth25pandey 
I think this is an important assignment as it was labeled. Because it is very useful for new geeks so, I would like you to review this.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,73 @@
-# Contributing to NeoAlgo
+# Contributing Guidlines
```
Comment (iamrajiv) [inline CONTRIBUTING.md]: @Rishikeshrajrxl fix typo mistake it should be `Guidelines`

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Rishikeshrajrxl format the file using markdown formatter and avoid using a lot of `<br>` there is unnecessary line breaks.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Rishikeshrajrxl Use Prettier to format the markdowns. You can use this https://prettier.io/playground/

**Comment by Rishikeshrajrxl** [comment]
@siddharth25pandey 
@iamrajiv 
@HarshCasper 
I have made all the necessary changes. Please do the review.

**Review by iamrajiv — APPROVED** [review]

**Comment by Rishikeshrajrxl** [comment]
@siddharth25pandey 
please review this.

**Comment by Rishikeshrajrxl** [comment]
@siddharth25pandey 
@HarshCasper 
guys, please review this. as soon as possible.

**Pull request merged**